Use full width on keyboard shortcut page

mjankowski opened the pull request:
Before:
<img width="622" alt="Screenshot 2024-09-26 at 12 54 16" src="https://github.com/user-attachments/assets/85a8c650-7a08-4be4-bc30-b92d001a856c">

After:
<img width="610" alt="Screenshot 2024-09-26 at 12 54 27" src="https://github.com/user-attachments/assets/c989d67e-67f4-4882-a27e-5f1e59baf64b">

Swap order of elements (label on left, key value on right), make table take up full space, style once over on `kbd`, etc.

mjankowski committed:
Use full width on keyboard shortcut page

oneiros commented:
I really like the new key design, that is a really nice touch.

I am not so sure about spacing things out in this way. It is really easy to lose track of which row you are in when trying to read this.

An often-used solution to this is adding a striped background, but I wonder, why not just keep the old order? That version did not have that problem imo.

mjankowski commented:
> I really like the new key design, that is a really nice touch.

Heavily inspired by the github shortcut popup thing, which has similar key look.

> I am not so sure about spacing things out in this way. It is really easy to lose track of which row you are in when trying to read this.

I don't disagree with this, but I will say that the screenshot I added is (I think) the widest possible version of this, and at ay smaller sizes it will collapse a bit so this does represent the worst case of losing tracking.

> An often-used solution to this is adding a striped background, but I wonder, why not just keep the old order? That version did not have that problem imo.

I changed it because in my own mental model of what is the label/header and what is the cell/data, it makes more sense to switch it, but only barely so. If this is otherwise acceptable, I can undo the order change portion.

mjankowski commented:
How can I push this forward?

Is this mergable if I flip the order back and do key style improvement only for now?
